dbms/src/Operators/UnorderedSourceOp.cpp

```diff
@@ -35,6 +36,16 @@ UnorderedSourceOp::UnorderedSourceOp(
 {
     setHeader(AddExtraTableIDColumnTransformAction::buildHeader(columns_to_read_, extra_table_id_index_));
     ref_no = task_pool->increaseUnorderedInputStreamRefCount();
+
+    if (is_disagg_)
+    {
+        const size_t connections = 2;
```

Comment:
One for send, the other for receive?

Reply:
one for inter connection, the other one for inner connection.